### PR TITLE
Stop building phone docs

### PR DESCRIPTION
**Pull request by nottrobin**
They are huge, and filling up our servers, but we're not even using them yet.

### CODE REVIEW

File: rebuild-docs [hunk 1]
```diff
@@ -3,13 +3,13 @@
 set -euo pipefail
 
 maas_dir=maas-docs
-phone_dir=phone-docs
 core_dir=ubuntu-core-docs
-snapcraft_dir=snapcraft-docs
+#snapcraft_dir=snapcraft-docs
+#phone_dir=phone-docs
```
Comment (anthonydillon) [inline rebuild-docs]: Isn't it bad practise to comment out code? These lines are in the repo history.

Comment (nottrobin) [inline rebuild-docs]: Fine.

Comment (nottrobin) [inline rebuild-docs]: @anthonydillon done

**Comment by nottrobin** [comment]
@anthonydillon care to review again?

**Pull request merged**